docs/components/InputText/Web.stories.tsx

```diff
@@ -12,7 +13,49 @@ export default {
 } as ComponentMeta<typeof InputText>;
 
 const BasicTemplate: ComponentStory<typeof InputText> = args => {
-  return <InputText {...args} />;
+  const [show, setShow] = React.useState(false);
```

Comment:
some example code that isn't needed. either remove entirely or make an example with this.